### ISSUE TITLE
Problems with boolean arguments

### ISSUE THREAD

**Issue by sciurius** [JavaWiz1/kodi-cli#4]
Trying to pass a boolean param fails. Even when the param is optional.

For example, the example from the docs:

    kodi-cli Application.SetMute mute=false

fails with
````
{
  "error": {
    "code": -32602,
    "data": {
      "method": "Application.SetMute",
      "stack": {
        "message": "Invalid type integer received",
        "name": "mute",
        "type": [
          "string",
          "boolean"
        ]
      }
    },
    "message": "Invalid params."
  },
  "id": 1,
  "jsonrpc": "2.0"
}

````

**Comment by JavaWiz1** [comment]
Looks like the help text has an issue.  That call ONLY takes a  string: enum [toggle] value.  It toggles mute on and off, there is no way to set mute on or off directly.

Syntax: kodi-cli Application.SetMute mute=toggle

The return value indicates whether the volume is muted (True) or un-muted (False)

I'll take a look at the code that generates the help and see why it shows that it accepts True/False as input when that is only for the output values....

**Comment by sciurius** [comment]
Allow me to re-open, the example shown is just one case. I have the same problem with other calls, e.g. 

````
kodi-cli Addons.ExecuteAddon addonid=plugin.video.isisplayer params=file=/some/file.mkv
{
  "error": {
    "code": -32602,
    "data": {
      "method": "Addons.ExecuteAddon",
      "stack": {
        "message": "Invalid type integer received",
        "name": "wait",
        "type": "boolean"
      }
    },
    "message": "Invalid params."
  },
  "id": 1,
  "jsonrpc": "2.0"
}
````

In fact, I haven't been able to use `Addons.ExecuteAddon` at all. Supplying `wait=false` bails out. Not supplying wait (it should default to false) also bails out.

**Comment by JavaWiz1** [comment]
I believe the above is fixed with v 0.2.2.
